convert_gvf_to_vcf/logger.py

```diff
@@ -7,7 +7,7 @@
 def set_up_logging(log_path=None):
     # Set up paths
     input_folder = os.path.dirname(__file__)
-    output_folder = os.path.join(input_folder, "..","tests", "output")
+    output_folder = os.path.normpath(os.path.join(input_folder, "..","tests", "output"))
     if log_path is None:
```

Comment:
Only noticing this now, but why is the code trying to write log to the tests folder ?